roles/dns/tasks/main.yml

```diff
@@ -0,0 +1,22 @@
+- name: Install Bind
+  action: "{{ ansible_pkg_mgr }} name=bind"
+
+- name: Configure Bind
+  template:
+    src: "{{ item.src }}"
+    dest: "{{ item.dest }}"
+    validate: "{{ item.validate }}"
+  with_items:
+    - src: openstack.zone
```

Comment:
I can see this role as being more generically applicable than just for openstack, could you add variables in place of the openstack specific references?